### DISCUSSION TITLE
Is there a step by step guide on setting up renovate for Bitbucket Cloud (free)?

### DISCUSSION THREAD

**Discussion by alvindizon** [renovatebot/renovate#18051] [Request Help] [answered]
Hello, I am trying to set up renovatebot for a repository in Bitbucket Cloud. I've read the README and it's pointing me to use the self-hosted option. I am confused, where would I run npm/docker?

Also I am going to try the method outlined here: https://dev.to/danielbayerlein/renovate-meets-bitbucket-pipelines-4ohk but it's not entirely clear to me as well. Has anyone here tried setting up renovatebot in Bitbucket?

**Comment by rarkins** [comment, marked as answer]
You would need to run Renovate via npm or Docker using a server or pipeline which you control. There currently isn't any hosted version, so you need to self-host.

Here's another example I found from a Google search: https://chihching.net/renovate-bitbucket-cloud
